packages/design-system/src/components/search/stories/input.js

```diff
@@ -30,6 +29,16 @@ import SearchInput from '../input';
 
 export default {
   title: 'DesignSystem/Components/Search/Input',
+  args: {
+    ariaClearLabel: 'label for clear button',
```

Comment:
🤔 Wondering if we need controls for aria labels + non visible updates. I think it's important that these are accessible but wondering if they need to be updatable through storybook.

Question for another time though 😄